Created the Project Panel Page

rsidhi088 opened the pull request:
The webpage specific to a single project has been created. This Page provides project and the graph details(yet to be included). An option has been provided to edit the project details if required. The branch pertaining to these changes is create-proj-panel-page.

pratima-shegokar committed:
Authentication and User creation.

Uses /register to create new user

Name, companyName, email and password are required fields

abdnaqeeb committed:
Added the base model entity class Project.java, and ProjectRepo.java with all notations required to link to the DB.

abdnaqeeb committed:
Added the project controller with all mapping notations to be accessed by frontend, and added the project service class which handles all functions deligated by the controller

abdnaqeeb committed:
Added the default constructor which has no parameters to Project.java

abdnaqeeb committed:
Added the Test for adding and geting all items from the data base for projects.

abdnaqeeb committed:
Merge branch 'backend-create-project-model' into backend-create-record-model

abdnaqeeb committed:
Added Record entity class with all fields and notations.

abdnaqeeb committed:
Added Record Repo interface with one more method findByProject.

abdnaqeeb committed:
Added Record Service with normal functionality of adding, updating, getting and deleting records, and added to it one more method getByProject which is reflected also in RecordRepo by the way.

abdnaqeeb committed:
added method getLastUpdatedOnProject in class RecordService to get the latest updated record in the project.

abdnaqeeb committed:
Added Record Controller class with normal functionality to deligate adding, updating, getting and deleting records to the Record Service, and added to it one more method getByProject which is reflected also in RecordService by the way.

abdnaqeeb committed:
changed the name of method getLastUpdatedOnProject to getLatestOnProject, and added some comments.

abdnaqeeb committed:
changed the GetMapping name of method getByProject from '/records' to '/records-project', and added new method getLatestOnProject.

abdnaqeeb committed:
Added method CreatePlannedValuesRecords but not completed yet.

abdnaqeeb committed:
changed the names of the columns in class Project by replacing the dash with underscore, and added a default constructor to the same class with no parameters.

rsidhi088 committed:
Merge pull request #20 from pratima-shegokar/login-backend

Authentication and User creation.

jshortz committed:
Merge pull request #22 from pratima-shegokar/backend-create-project-model

Backend create project model

jshortz committed:
Added a very minimum Why Perflow page. Added link to navbar. Moved MyProject.js into appropriate folder for code structure. Edited comments in Navbar.js for minor grammar

jshortz committed:
Added the PerFlow background image and a sample graph (purely placeholder - not ours). Updated App.css to style WhyPerflow.js. Added info and fake testimonials to WhyPerflow.js

pratima-shegokar committed:
Merge pull request #40 from pratima-shegokar/FE-Why-Perflow-Page-create

Added a very minimum Why Perflow page. Added link to navbar. Moved My…

pratima-shegokar committed:
Render login page to be displayed on the first page

pratima-shegokar committed:
Add route for login to be displayed only when clicked login.

pratima-shegokar committed:
Renamed to login to avoid homepage confusion

jshortz committed:
Combined frontend README with main README. Deleted FE README.

abdnaqeeb committed:
just done the same adjustment done for the breviously pushed branch for project model, so it will not conflict with the master.

abdnaqeeb committed:
Added the method calcCPVList in RecordService class to calculate the cumulative planned value list from the project info.

abdnaqeeb committed:
Added the method calcCPCIFList in RecordService class to calculate the cumulative planned cash inflow list from the project info and cumulative planned value list.

abdnaqeeb committed:
Added a test of the basic functionality of calcCPVList method in RecordService class.

abdnaqeeb committed:
Added a test of the basic functionality of calcCPCIFList method in RecordService class.

abdnaqeeb committed:
Merge branch 'master' into backend-create-record-model

jshortz committed:
Delete robotstxt.html that no one knew the purpose of

abdnaqeeb committed:
added /bin folder to .gitignore.

MelanyCristy reviewed (commented):
Hey! You should fix the conflicts first and then create a pull request. This pull request contains changes and commits already done/merged in master

pratima-shegokar committed:
Completed rendering login form and also add logo on login page.

abdnaqeeb committed:
Add method RecordService.createPlannedValuesRecords and modified the methods names calcCPvList to  calcPvList and calcCPCIFList to calcPcifList.

abdnaqeeb committed:
changed the name of local vaiable in method RecordService.calcPvList from cPvList to pvList.

abdnaqeeb committed:
changed the name of local variables in method RecordService.calcPcifList from cPCIFList to

abdnaqeeb committed:
Added a test for the method RecordService.createPlannedValuesRecords

abdnaqeeb committed:
Added method RecordService.addList but not tested yet

abdnaqeeb committed:
added the method RecordService.addNewProjectRecords.

abdnaqeeb committed:
change the RecordService to be public, and addNewProjectRecords method to be public, so we can invoke it from ProjectController.

jshortz committed:
Set up router in App.js for Home. Created 2 intro paragraphs and 1 register button. Set background color.

abdnaqeeb committed:
defined the RecordService as autowired in ProjectController, and modified the add method in ProjectController to invoke the addNewProjectRecords from RecordsService.

abdnaqeeb committed:
modified method RecordService.addNewProjectRecords to assign the project for all records.

jshortz committed:
Changed background color of page and text color. Added link to registration page for button.

jshortz committed:
Changed register button color. Removed link decoration on register button.

jshortz committed:
Merge pull request #42 from pratima-shegokar/FS-code-restructure

Combined frontend README with main README. Deleted FE README.

jshortz committed:
Removed unusued line in App.css related to text decoration

jshortz committed:
Made navbar transparent

pratima-shegokar committed:
Call backend and authenticate

pratima-shegokar committed:
Added axios library

pratima-shegokar committed:
Added rendering of registration form

pratima-shegokar committed:
Use api for authentication

Use new navbar once loggedIn

Updated App.js with 2 routers for loggedin and loggedout state

pratima-shegokar committed:
remove sign up link from navbar

pratima-shegokar committed:
Merge pull request #44 from pratima-shegokar/backend-create-record-model

Backend create record model

jshortz committed:
Updated registration link on homepage

jshortz committed:
Removed unusued function from Home.js

pratima-shegokar committed:
Merge branch 'create-proj-panel-page' into master

jshortz committed:
Update LoginForm.js

MiladYaqo committed:
Merge branch 'master' into FE-Home-Page-create

MiladYaqo committed:
Merge pull request #45 from pratima-shegokar/FE-Home-Page-create

Fe home page create, merge is confirmed and the conflicts were resolved, please do resolve them next time.

jshortz committed:
Merge branch 'master' into login-frontend

jshortz committed:
Merge branch 'master' into login-frontend

jshortz committed:
Merge pull request #47 from pratima-shegokar/login-frontend

Login frontend

jshortz committed:
Merge pull request #46 from pratima-shegokar/FE-Navbar-Update

Made navbar transparent
